searvey/usgs.py

```diff
@@ -279,7 +279,7 @@ def get_usgs_station_data(
         while rate_limit.reached(identifier="USGS"):
             wait()
 
-    endtime = resolve_date(endtime)
+    endtime = resolve_timestamp(endtime, timezone_aware=False).date()
```

Comment:
@SorooshMani-NOAA AFAI can tell dataretrieval/USGS handle the time component just fine. For example:
``` python
import dataretrieval.nwis as nwis

print(len(nwis.get_record(sites='03339000', service='iv', start='2017-12-31', end='2018-01-01')))  # prints 192
print(len(nwis.get_record(sites='03339000', service='iv', start='2017-12-31T12:00:00', end='2018-01-01')))  # prints 144
```
In that sense we don't **need** to call `.date()` in line 282. I only added that in order to keep the old behavior and to let the tests pass. Your call if we keep it as is or we remove `.date()`.

Reply:
@pmav99 let's keep the behavior the same as before by adding `.date()` for now. Thanks for notifying me.